src/Admin/TimelineMeasureAdmin.php

```diff
@@ -23,14 +24,41 @@ public function __construct($code, $class, $baseControllerName, MeasureManager $
         $this->measureManager = $measureManager;
     }
 
+    public function createQuery($context = 'list')
+    {
+        $query = parent::createQuery();
+        $alias = $query->getRootAlias();
+
+        $query
+            ->leftJoin("$alias.translations", 'translations')
+            ->addSelect('translations')
+            ->leftJoin("$alias.themes", 'themes')
+            ->addSelect('themes')
+            ->leftJoin("$alias.profiles", 'profiles')
+            ->addSelect('profiles')
+        ;
+
+        return $query;
+    }
+
     protected function configureFormFields(FormMapper $formMapper)
     {
         $formMapper
-            ->with('Méta-données', ['class' => 'col-md-6'])
-                ->add('title', TextType::class, [
-                    'label' => 'Titre',
-                    'filter_emojis' => true,
+            ->with('Traductions', ['class' => 'col-md-6'])
+                ->add('translations', TranslationsType::class, [
+                    'by_reference' => false,
+                    'label' => false,
+                    'default_locale' => ['fr'],
+                    'locales' => ['fr', 'en'],
+                    //'required_locales' => ['fr', 'en'],
```

Comment:
Please remove or uncomment, or add a reason why this is commented.

Reply:
will remove it